README.md

```diff
@@ -844,6 +845,9 @@ The following example of a proportion boolean measure shows how the logic highli
 
 ![Screenshot of Highlighting HTML Measure Logic](./static/logic-highlighting-example-2.png)
 
+### CQL Statement Ordering
```

Comment:
```suggestion
### CQL Statement Ordering in HTML
```
Maybe this is unnecessary, but I am find the mixed use of "HTML Ordering" and "CQL Statement Ordering" a bit confusing and maybe this will clear it up a bit without losing any important info?